http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModelSpec.scala

```diff
@@ -148,14 +151,22 @@ class LeonardoModelSpec extends LeonardoTestSuite with AnyFlatSpecLike {
 
   it should "create UserScriptPath objects according to provided path" in {
     val gcsPath = "gs://userscript_bucket/userscript.sh"
-    val httpPath = "https://userscript_path"
+    val httpPath = "https://w3.org"
```

Comment:
just curious why the change :)

Reply:
The other validator didn't like the previous input, I think in part because it's not truly a valid URL